Xamarin.Forms.Platform.iOS/Material/MaterialSliderRenderer.cs

```diff
@@ -0,0 +1,183 @@
+using System;
+using System.ComponentModel;
+using CoreGraphics;
+using Foundation;
+using MaterialComponents;
+using UIKit;
+using Xamarin.Forms;
+using Xamarin.Forms.PlatformConfiguration.iOSSpecific;
+using MSlider = MaterialComponents.Slider;
+using Specifics = Xamarin.Forms.PlatformConfiguration.iOSSpecific.Slider;
+
+[assembly: ExportRenderer(typeof(Xamarin.Forms.Slider), typeof(Xamarin.Forms.Platform.iOS.Material.MaterialSliderRenderer), new[] { typeof(VisualRendererMarker.Material) })]
+
+namespace Xamarin.Forms.Platform.iOS.Material
+{
+	public class MaterialSliderRenderer : ViewRenderer<Slider, MSlider>,
+		ISliderDelegate
```

Comment:
This should actually be removed if we don't want to disable the touch/jump action.